spec/support/test_environment.rb

```diff
@@ -6,6 +6,7 @@
   config.include RspecTagMatchers
   config.include CustomMacros
   config.mock_with :rspec
+  config.deprecation_stream = '/dev/null'
```

Comment:
It's only RSpec deprecation stream, not Rails. While I don't like this either, the test suite generates a ton of deprecations because of old should syntax and other things.